src/utils/constants.rs

```diff
@@ -1,6 +1,10 @@
 pub const MADARA_REPO_ORG: &str = "keep-starknet-strange";
 pub const MADARA_REPO_NAME: &str = "madara";
 
+pub const KARNOT_REPO_ORG: &str = "karnotxyz";
+
+pub const EXPLORER_COMPATIBLE_KARNOT_MADARA_BRANCH: &str = "custom_events_page";
```

Comment:
let's just call it `BRANCH_NAME`, there could be other reasons to be on a different branch